src/main/java/org/powerflows/dmn/engine/model/decision/field/Output.java

```diff
@@ -30,7 +28,6 @@
     private String name;
     private String description;
     private ValueType type;
-    private Expression expression;
```

Comment:
Why did you remove expression from output?

Reply:
output declaration doesnt support it, its evaluated, it only makes sense to have expression in output entry inside rule